src/dlgTriggerEditor.h

```diff
@@ -46,14 +46,18 @@
 #include "dlgTimersMainArea.h"
 #include "dlgTriggersMainArea.h"
 #include "dlgVarsMainArea.h"
-#include "SingleLineTextEdit.h"
+#include "dlgTriggerEditorCommand.h"
 
 #include "pre_guard.h"
 #include <QDialog>
+#include <QUndoStack>
+#include <QUndoView>
+#include <QUndoCommand>
 #include <QFlag>
 #include <QListWidgetItem>
 #include <QScrollArea>
 #include <QTreeWidget>
+#include <QDockWidget>
 #include "post_guard.h"
```

Comment:
Sort them if possible - it helps when tracking additions and removals...
```suggestion
#include "pre_guard.h"
#include <QDialog>
#include <QDockWidget>
#include <QFlag>
#include <QListWidgetItem>
#include <QScrollArea>
#include <QTreeWidget>
#include <QUndoCommand>
#include <QUndoStack>
#include <QUndoView>
#include "post_guard.h"
```